site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/repository/EntityStore.java

```diff
@@ -11,4 +13,6 @@
      * @return the entity stored id.
      */
     String saveEntity(T entity);
+
+    List<T> geAllEntities();
```

Comment:
1. Typo in the method name.
2. Some of us had a discussion on the naming conventions of methods in the repository layer. Get has a "client" sound to it, but since a repository is similar to a db maybe call it something like "fetch". A question in our internal channel might be good too...

Reply:
Changed to findAll() to match spring-data usage (and findSites() in the SiteService).